cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -419,17 +425,18 @@ def setup_monitoring(self, targets):
                                            port_mappings=[
                                                ecs.PortMapping(container_port=9091)]
                                            )
-
         c_postgres = task.add_container('postgres',
                                         essential=True,
                                         image=ecs.ContainerImage.from_registry(
-                                            'ghcr.io/emqx/perf-test-postgres:master'),
+                                            'ghcr.io/k32/sysmon-postgres:0.1.1'),
                                         port_mappings=[
                                             ecs.PortMapping(container_port=5432)],
                                         stop_timeout=Duration.seconds(10), # It looks like postgres doesn't want to die sometimes
                                         start_timeout=Duration.seconds(300),
                                         environment={
-                                            'POSTGRES_PASSWORD': '123',
+                                            'POSTGRES_PASSWORD': self.postgresPass,
+                                            'SYSMON_PASS': self.postgresPass,
```

Comment:
Password reuse is a bit confusing 🤔

Reply:
Having different passwords for the different users allows to give grafana only a read-only access, and restrict access for system_monitor. This all makes sense in more sophisticated environments when monitoring setup and the nodes are controlled by different parties. This is not the case in our test environment.